src/news/2021-07-06-Reducing-Barriers-to-Virtual-Health.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reducing Barriers to Virtual Health
+filename: Reducing-Barriers-to-Virtual-Health
+date: '2021-07-06'
+---
+
+The IUI project team at the IDRC is pleased to share the [Guide for Reducing Barriers to Virtual Health](https://wiki.fluidproject.org/display/IUIGFVP/Guide+for+Reducing+Barriers+to+Virtual+Healthcare)
+and its accompanying [Personal Preferences Approach to personalizing healthcare](https://wiki.fluidproject.org/display/IUIGFVP/Section+-+Software+Approaches+for+Expressing+Personal+Preferences+for+Healthcare).
+
+The guide has two main purposes: 1) to identify the virtual healthcare needs of Canadians and 2) to provide guidance for
+reducing the barriers in receiving care. The guide incorporates the input and collaboration of co-design participants of
+various lived experiences and backgrounds, and healthcare practitioners from across Canada. It also includes insights
+from literature research and our collective experiences in Inclusive Design. For more information on the methodology
+used to create this initial version of the guide, please visit this project wiki page: [Methodology used to create the Guide](https://wiki.fluidproject.org/display/IUIGFVP/Methodology+used+to+create+the+Guide).
+
+The Personal Preferences Approach explores a system that allows for the expression of personal preferences for virtual
+healthcare and responding to these preferences in a meaningful way that reduces barriers. This system is informed by the
+principles, practices, and approaches from the Guide for Reducing Barriers to Virtual Healthcare and builds on previous
+work for Learner Options. Although this approach is contextualized for healthcare, the ideas can be applied to other
```

Comment:
I feel like the term "Learner Options" should link to somewhere for more information. Any suggestions?